core/modes/modes_not_dragon.talon

```diff
@@ -0,0 +1,32 @@
+mode: sleep
+mode: command
+mode: dictation
+not speech.engine: dragon
+-
+^talon wake [<phrase>]$: speech.enable()
+
+# We define this *only* if the speech engine isn't Dragon, because if you're using Dragon,
+# "wake up" is used to specifically control Dragon, and not affect Talon.
+#
+# It's a useful and well known command, though, so if you're using any other speech
+# engine, this controls Talon.
+^(wake up)+$: speech.enable()
```

Comment:
how come this one doesn't have an optional phrase like the others? too common an utterance?

Reply:
couldn't tell ya, I only toggle speech via a button these days

Reply:
Yeah, I didn't change this:
https://github.com/talonhub/community/blob/main/core/modes/sleep_mode_not_dragon.talon

I don't really see the use case for phrase with wake up tbh

Reply:
oh yeah good point